packages/server/tsconfig.json

```diff
@@ -29,6 +29,6 @@
         "name": "Kaetram",
         "hideGenerator": true
     },
-    "files": ["ts/main.ts"],
+    "includes": ["ts/**/*.ts", "data/**/*.json"],
```

Comment:
I believe this is not needed as we got rid of dynamic imports here https://github.com/Kaetram/Kaetram-Open/pull/42

Reply:
Alright.  That works too.  This way would work for both but can remove

Reply:
OK, let's remove this change